test/functional/outputs/cloudwatch/forward_to_cloudwatch_test.go

```diff
@@ -64,7 +64,7 @@ var _ = Describe("[Functional][Outputs][CloudWatch] Forward Output to CloudWatch
 		defaultRetryInterval, _ = time.ParseDuration("10s")
 
 		getRawCloudwatchApplicationLogs = func(f *functional.CollectorFunctionalFramework, cwlClient *cwl.Client) ([]string, error) {
-			log.V(3).Info("Retrieving cloudwatch LogGroups")
+			log.V(2).Info("Retrieving cloudwatch LogGroups")
```

Comment:
is this functional test enabled for vector?

Reply:
this variable only setup the collector as vector. this variable does not choose which tests will run under `make test-functional-vector`